scripts/caput-pipeline

```diff
@@ -13,8 +13,12 @@ products = None
 
 
 @arg('configfile', help='Configuration file to run.')
-def run(configfile):
+@arg('--loglevel', help='Logging level (DEBUG, INFO, WARNING, ERROR, CRITICAL)')
+def run(configfile, loglevel='INFO'):
     from caput.pipeline import Manager
+    import logging
+    level = getattr(logging, loglevel.upper())
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Doesn't this just always set the log level to `INFO`?

Reply:
I must have been drunk when I wrote this. I'll fix it.